### PR TITLE
zh-cn: update the translation of `Element.setCapture()`

### LINKED ISSUE #16955: doc of 'element.setCapture' in in Chinese missing some tips
### MDN URL

https://developer.mozilla.org/zh-CN/docs/Web/API/Element/setCapture

### 与此议题相关的小节或标题？

_No response_

### 哪些信息是错误、无用或不完善的？

Missing 'Deprecated' and 'Non-standard' tips.

### 你期望看到什么？

Show 'Deprecated' and 'Non-standard' tips.

### 你有什么支持的链接、参考或引用？

https://developer.mozilla.org/zh-CN/docs/Web/API/Element/setCapture

### 你还想补充什么内容？

_No response_

### MDN metadata

<details>
<summary>Page report details</summary>

* Folder: `zh-cn/web/api/element/setcapture`
* MDN URL: https://developer.mozilla.org/zh-CN/docs/Web/API/Element/setCapture
* GitHub URL: https://github.com/mdn/translated-content/blob/main/files/zh-cn/web/api/element/setcapture/index.md
* Last commit: https://github.com/mdn/translated-content/commit/2748fdf6b9cac93e9fd97ca2c1d6d9fa272faaab
* Document last modified: 2023-07-30T02:54:27.000Z

</details>

### PR DESCRIPTION
**Pull request by yin1999**
### Description

update the translation of `Element.setCapture()`

### Related issues and pull requests

Fixes: #16955

### CODE REVIEW

File: files/zh-cn/web/api/element/setcapture/index.md [hunk 1]
```diff
@@ -52,33 +68,34 @@ element.setCapture(retargetToElement);
...
-    </p>
-    <div id="output">还没有任何事件哦！</div>
+    <p><a id="myButton" href="#">在这里测试</a></p>
+    <div id="output">还没有任何事件</div>
   </body>
 </html>
 ```
 
-[查看在线演示](/samples/domref/mousecapture.html)
+[查看在线演示](https://mdn.dev/archives/media/samples/domref/mousecapture.html)
```
Comment (jasonren0403) [inline files/zh-cn/web/api/element/setcapture/index.md:84]: Does this example still work now? Maybe we should remove it. 🤔

Comment (yin1999) [reply files/zh-cn/web/api/element/setcapture/index.md:84]: Only works in firefox (According to the bcd, I'd like to test it later today)

Comment (yin1999) [reply files/zh-cn/web/api/element/setcapture/index.md:84]: It works in firefox

image

**Review by jasonren0403 — APPROVED** [review]
LGTM 👍

**Pull request merged**